src/actions/authActions.jsx

```diff
@@ -18,7 +18,7 @@ const addToFirebaseDB = (
         imageURL,
         email,
         provider,
-        userType: "normal",
+        userType: "Subscriber",
```

Comment:
as mentioned any enum types that are used in firebase should be lowercase or uppercase (mixed case is the most prone to errors, in case you ever do "userType === ??" for comparison all caps or no caps ensures minimal chancce of bugs

Reply:
Actually this is for another PR because I already made these changes